### DISCUSSION TITLE
Trackers overlap in the Map Card

### DISCUSSION THREAD

**Discussion by schaefsteven** [home-assistant/frontend#17985] [Dashboard]
In the Map Card, if two trackers are right on top of each other, (If my partner and I are both at home,) Then one will be on top and you can't see the other. 

In many cases it's not a big deal and you can infer the information, but I think it would be better to have some kind of UI element that merges the two trackers together if the zoom level of the map doesn't allow you to see both of them.

**Comment by schaefsteven** [comment]
Did a little looking around, and it looks like there is a plugin for leaflet.js that would provide the needed functionality: https://github.com/Leaflet/Leaflet.markercluster

**Comment by ildar170975** [comment]
I was thinking about a similar functionality (https://github.com/home-assistant/frontend/discussions/20724).
Kind of connecting markers by lines if they have same coords:
![327911530-d98c4e60-59ef-47d9-bb07-b1bd83071b76](https://github.com/home-assistant/frontend/assets/71872483/83ab29f2-7fa7-4ca3-ba22-6bd6a483ad71)

**Comment by ildar170975** [comment]
Will be hopefully solved with https://github.com/home-assistant/frontend/discussions/24198

**Comment by ildar170975** [comment]
Can be closed imho after https://github.com/home-assistant/frontend/discussions/24198
